metis-enrichment/metis-enrichment-service/src/main/java/eu/europeana/enrichment/service/EntityService.java

```diff
@@ -43,18 +53,27 @@ public OrganizationTermList storeOrganization(Organization org) {
 
 		// store term list
 		return (OrganizationTermList) MongoDatabaseUtils
-				.insertMongoTermList(termList);
+				.storeMongoTermList(termList);
+
 	}
 
 	private OrganizationTermList organizationToOrganizationTermList(
-			OrganizationImpl organization) {
+			OrganizationImpl organization, Date created, Date modified) {
 		OrganizationTermList termList = new OrganizationTermList();
 		if (organization.getPrefLabel() == null
```

Comment:
StringUtils.isEmpty() can be used.

Reply:
actually not, prefLable is a map. But I change to use null || isEmpty()

organization.getPrefLabel() == null
				|| organization.getPrefLabel().isEmpty()